stingray/crossspectrum.py

```diff
@@ -1238,13 +1238,16 @@ def time_lag(self):
             return ph_lag / (2 * np.pi * self.freq)
         else:
             raise AttributeError("Object has no attribute named 'time_lag' !")
-
-    def plot(self, labels=None, axis=None, title=None, marker="-", save=False, filename=None):
+    
+    def plot(self, axes =None, labels=None, axis=None, title=None, marker="-", save=False, filename=None):
```

Comment:
This new keyword should be added at the end of the list, in case someone used the other keywords in order.